tests/components/file_upload/test_init.py

```diff
@@ -64,3 +64,42 @@ async def test_removed_on_stop(hass: HomeAssistant, hass_client, uploaded_file_d
 
     # Test it's removed
     assert not uploaded_file_dir.exists()
+
+
+async def test_upload_large_file(hass: HomeAssistant, hass_client, large_file):
+    """Test uploading large file."""
+    assert await async_setup_component(hass, "file_upload", {})
+    client = await hass_client()
+
+    with patch(
+        # Patch temp dir name to avoid tests fail running in parallel
+        "homeassistant.components.file_upload.TEMP_DIR_NAME",
+        file_upload.TEMP_DIR_NAME + f"-{getrandbits(10):03x}",
+    ), open(large_file.name) as fp:
```

Comment:
instead of using an actual file to upload, can we just pass a TextIO https://docs.python.org/3/library/io.html#text-i-o

Reply:
Oh upps, you mentioned that earlier, should be fine now!